src/ctl.rs

```diff
@@ -69,6 +69,46 @@ pub mod set_cursor {
     }
 }
 
+/// Get cursor position
+pub mod cursor_pos {
```

Comment:
This already exists, also `unwrap` & `expect` are banned in this codebase, no exceptions
https://github.com/hyprland-community/hyprland-rs/blob/f0fe365249e724dc0d4a1db5c7100d734dc7ac9f/src/data/regular.rs#L405